### ISSUE TITLE
In Settings, Sign in to Iridium does not work.

### ISSUE THREAD

**Issue by Ornataweaver** [iridium-browser/iridium-browser-osx#23]
Maybe it's because of two factor authentication of google.

**Comment by tinfoil920-39io** [comment]
You cannot value privacy and also use "free" google services.  Offering yourself as chattel is opposed to Human Rights.

**Comment by Ornataweaver** [comment]
Then remove the option from the settings.
Iridium is buggy and is not being maintained. I abandoned it long ago. I couldn't even remove the extensions I had with the current version. I had to open an older version to deactivate extensions I didn't want.

**Comment by medozas** [comment]
@Ornataweaver : We're checking the possibilities to disable the option. Onfortunately, Google is making it quite hard to extract certain functions - Not having checked this specific case. Nevertheless, I have no idea how you get to the conclusion that it is not maintained ... I'd take a look at https://iridiumbrowser.de/news/ before issuing such statements... And for extensions: Do you have more details?

**Comment by Ornataweaver** [comment]
I didn't mean to insult you spending your precious time on this project. I reached the conclusion because I reported the issues so long ago and didn't receive any replies for long.
About the extensions bug, I could add new extensions via chrome store. But couldn't disable them or remove them in extensions settings. I had to use an older version for that purpose. I don't remember the version since I have deleted it from my computer. sorry for that.
